src/common/Error/styled.js

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+
+export const ErrorWrapper = styled.article`
+  margin-top: 96px;
+  display: grid;
+  justify-items: center;
+  grid-gap: 32px;
+  text-align: center;
+  color: ${({ theme }) => theme.colors.mineShaft};
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.tablet}px) {
+    grid-gap: 24px;
+  }
+`;
+
+export const Header = styled.header`
+  font-weight: 700;
+  font-size: 36px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.tablet}px) {
+    font-size: 16px;
+  }
+`;
+
+export const Paragraph = styled.p`
```

Comment:
In Figma project there are another style added to this element:
font-weight: 500;
font-size: 22px;

Reply:
In this example font-weight doesn't make any difference that's a reason why I didn't add it to the selector but definitely outline semantically is needed. I didn't add it but I wanted to mention it and ask you what do you think girls about the outline so great that you are saying it. I think Krystian can do it additionally because you have to discus about the styles - what kind of etc.